scripts/lint-lists.py

```diff
@@ -92,6 +95,15 @@ def load_categories(path, get_description_code=get_new_description_code):
             code_map[code] = desc
     return code_map
 
+def load_global_list(path):
+    check_list = {}
+    with open(path, 'rb') as in_file:
+        reader = csv.reader(in_file, delimiter=',')
+        for idx, row in enumerate(reader):
+            if idx != 0 and (len(row) == 6):
+                check_list[row[0]] = row[1]
+    return check_list
```

Comment:
You can probably just use a `set` here instead of `dict`, since you don't need the value.